LayoutTests/fast/mediastream/device-change-event-4.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Testing 'devicechange' event is fired correctly.</title>
```

Comment:
Nit: it would be nice to have more descriptive, or at least different, titles for the tests

Reply:
Done, I removed this one given there is only one promise_test with a precise test name